providers/flagd/README.md

```diff
@@ -146,7 +146,7 @@ By default, the provider is configured to
 use [least recently used (lru)](https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/map/LRUMap.html)
 caching with up to 1000 entries.
 
-### OpenTelemetry support (RPC only)
+### OpenTelemetry tracing (RPC only)
 
 OpenTelemetry support can be enabled either through [automatic instrumentation](https://opentelemetry.io/docs/instrumentation/java/automatic/) 
 or with [manual instrumentation](https://opentelemetry.io/docs/instrumentation/java/manual/). 
```

Comment:
reading this is a bit misleading 🤔 
it is true that automatic instrumentation will produce gRPC spans, but it doesn't provide the attribute infos that we expose through the manual instrumentation

Reply:
We still get traces from automatic instrumentation. See this code [1]. As long as we have a tracer, span will be created and exported. 

[1] - https://github.com/open-feature/java-sdk-contrib/blob/main/providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/grpc/strategy/TracedResolving.java#L28-L35

Reply:
The problem that I have with how the whole paragraph is written is that it seems I can rely only on auto-instrumentation. This to me sounds like I don't need to provide **any** configuration to the provider about OTel. However, this would lead to the FlagD Provider using the `SimpleResolving` which doesn't augment traces.

Reply:
I updated this section of the documentation. Please check and let me know if this clears the doubts :)  - https://github.com/open-feature/java-sdk-contrib/blob/b0d3bb02ee5809db8f62cf22521e74f2bcdb4830/providers/flagd/README.md#opentelemetry-tracing-rpc-only

Reply:
Thanks a lot! ✅